utils/navigator_tools/nodes/tf_to_gazebo.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+import roslib
+import rospy
+import math
+import tf
+import geometry_msgs.msg
+import numpy as np
+def main():
```

Comment:
Thats what happens without the `if __name__ == "__main__"` tag, the call to the main function doesn't do anything different.